### PR TITLE
LineSplitWithVoltageLevel migration to react hook form and yup

### PR DESCRIPTION
**Pull request by ayolab**

### CODE REVIEW

File: src/components/refactor/dialogs/line-split-with-voltage-level/line-split-with-voltage-level-dialog.js [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+                lineSplit[CONNECTIVITY]?.[VOLTAGE_LEVEL]?.[ID],
+                lineSplit[CONNECTIVITY]?.[BUS_OR_BUSBAR_SECTION]?.[ID],
+                lineSplit[LINE1_ID],
+                sanitizeString(lineSplit[LINE1_NAME]),
+                lineSplit[LINE2_ID],
+                sanitizeString(lineSplit[LINE2_NAME])
+            ).catch((error) => {
+                snackError({
+                    messageTxt: error.message,
+                    headerId: 'ShuntCompensatorCreationError',
```
Comment (dbraquart) [inline src/components/refactor/dialogs/line-split-with-voltage-level/line-split-with-voltage-level-dialog.js:133]: this is not the correct error message

**Review by AbdelHedhili — COMMENTED** [review]
Is there a reason why you didn't delete the previous line-split form ?

**Comment by AbdelHedhili** [comment]
test : ok
console checks : ok
code : ok

Just need to delete the old form (unless there is a reason not to) and it's fine

File: src/components/refactor/dialogs/line-split-with-voltage-level/line-split-with-voltage-level-dialog.js [hunk 2]
```diff
@@ -0,0 +1,223 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
```
Comment (AbdelHedhili) [inline src/components/refactor/dialogs/line-split-with-voltage-level/line-split-with-voltage-level-dialog.js:2]: 2023

File: src/components/refactor/dialogs/line-split-with-voltage-level/line-split-with-voltage-level-form.js [hunk 3]
```diff
@@ -0,0 +1,162 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
```
Comment (AbdelHedhili) [inline src/components/refactor/dialogs/line-split-with-voltage-level/line-split-with-voltage-level-form.js:2]: 2023

**Pull request merged**